src/yb/client/client-internal.cc

```diff
@@ -2621,6 +2622,37 @@ Result<bool> YBClient::Data::CheckIfPitrActive(CoarseTimePoint deadline) {
   return resp.is_pitr_active();
 }
 
+std::vector<std::string> YBClient::Data::ActiveUniverseHistory() {
```

Comment:
What is the use-case for this function?

Reply:
I followed the other examples and they seemed to call into client-internal, so I followed the same pattern. But you are right, this can be done from client.cc itself.